app/perfunwind.cpp

```diff
@@ -445,6 +446,7 @@ void PerfUnwind::resolveCallchain()
         symbols->attachDwfl(&m_currentUnwind);
         m_currentUnwind.frames.append(symbols->lookupFrame(ip, isKernel,
                                             &m_currentUnwind.isInterworking));
+        m_currentUnwind.ipStack.append({ip, m_currentUnwind.ipStack.length()});
```

Comment:
github doesn't allow me to comment on non-changed lines, but see above in `PerfUnwind::unwindStack`, there `frames` is potentially mutated which will break the ipStack - can we accomodate for that or at least disable the ipStack for `isInterworking`?

Reply:
I disabled it, seems to be the lesser of two evils